packages/react/src/compatible-components/radio-group/radio-group.tsx

```diff
@@ -43,19 +60,37 @@ export function RadioGroupCompatible<T>({
     }
     return getItemLabel(item);
   };
+
+  const cssStyle = {
+    ...style,
+    width: otherProps.width || '',
+    height: otherProps.height || '',
+  };
+
   return (
-    <RadioGroup<T> {...radioGroupProps}>
-      {items.map((item, index) => {
-        const value = getItemValue(item);
-        const key = `${value}-${index}`;
-        return (
-          <RadioButton
-            key={key}
-            value={value}
-            label={renderLabel(item, index)}
-          />
-        );
-      })}
-    </RadioGroup>
+    (otherProps.visible ?? true)
+      ? (
+        <RadioGroup<T>
+          {...otherProps}
```

Comment:
Suggestion: Introduce new restProps and use it here instead of otherProps, and replace all props.* to local destructed variable.
```
const { width, height, accessKey, onFosusIn, onFocusOut, ...restProps} = otherProps;
```

Reply:
Yes, agree with you, I will fix it